packages/handshake-api/src/index.js

```diff
@@ -22,6 +22,7 @@ const cache = new NodeCache({ stdTTL: 300 }); // cache for 5 minutes
 
 // Match both `sia://HASH` and `HASH` links.
 const startsWithSkylinkRegExp = /^(sia:\/\/)?[a-zA-Z0-9_-]{46}/;
+const registryEntryRegExp = /^skydb:\/\/(?<publickey>[a-zA-Z0-9%]+)\/(?<datakey>[a-zA-Z0-9%]+)$/;
```

Comment:
What does the `%` do?

Reply:
public key looks like `"ed25519:d611373052bb124f23c3376a181e140390d9545666dae333acee2402dfc95f4a"`
uri-encoded looks like `"ed25519%3Ad611373052bb124f23c3376a181e140390d9545666dae333acee2402dfc95f4a"`

so we need to include `%` in character list

Reply:
Oh, I see. Thanks.